kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDownloadPanel.js

```diff
@@ -187,10 +184,10 @@ define([
                         app_id: method,
                     },
                 ])
-            ).then(([jobId]) => [jobId, wsObjectName]);
```

Comment:
No longer pass around the object name to different things. Previously, it was only injected into the download URL, blobstore doesn't do that.